silx/gui/dialog/ColormapDialog.py

```diff
@@ -828,7 +828,7 @@ def __init__(self, parent=None, title="Colormap Dialog"):
         self._item = None
         """Weak ref to an external item"""
 
-        self._colormapChange = _ColormapChanged()
+        self._colormapChange = utils.LockReentrant()
```

Comment:
Do you need to use LockReentrant elsewhere?

Reply:
Yes, in this PR or the next one i think, in the ROIs.